itou/utils/models.py

```diff
@@ -79,3 +79,24 @@ class PkSupportRemark(AbstractSupportRemark):
 
 class UUIDSupportRemark(AbstractSupportRemark):
     object_id = models.UUIDField()
+
+
+def check_date_order_constraint(
```

Comment:
Vu le contenu du check, il me semble peu approprié si les dates ne peuvent pas être `null`.
```suggestion
def check_nullable_date_order_constraint(
```